test/unit/src/math/Vector3.tests.js

```diff
@@ -745,6 +745,21 @@ export default QUnit.module( 'Maths', () => {
 
 		} );
 
+		QUnit.test( 'nonLinearLerp', ( assert ) => {
```

Comment:
Sweep: Can you add another test to check Vector3.Lerp is the same as Vector3.nonLinearLerp with a strength of 0.5?